.classpath

```diff
@@ -10,6 +10,7 @@
 		<attributes>
 			<attribute name="optional" value="true"/>
 			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="test" value="true"/>
```

Comment:
Minor nit, but this seems to be an IDE-generated file, and should probably be git-ignored.

Reply:
It's from back when I used Eclipse to develop this library, and it made sense to have the classpath already configured. Probably doesn't make much sense now (this update was automatically performed by `mvn` AFAICT), but that can happen in a later PR.